src/main/java/com/example/stockApi/serviceImpl/OrderServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+package com.example.stockApi.serviceImpl;
+
+import java.util.Optional;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.example.stockApi.dto.OrderRequestDto;
+import com.example.stockApi.dto.OrderResponseDto;
+import com.example.stockApi.entity.Account;
+import com.example.stockApi.entity.AmountOfStock;
+import com.example.stockApi.entity.OrderEntity;
+import com.example.stockApi.enums.OrderType;
+import com.example.stockApi.repository.AccountRepository;
+import com.example.stockApi.repository.AmountOfStockRepository;
+import com.example.stockApi.repository.OrderRepository;
+import com.example.stockApi.service.OrderService;
+
+@Service
+public class OrderServiceImpl implements OrderService{
+	
+	private OrderRepository orderRepository;
+	private final AccountRepository accountRepository;
+	private final AmountOfStockRepository amountOfStockRepository;
+	
+    public OrderServiceImpl(OrderRepository orderRepository,
+    		AccountRepository accountRepository,
+            AmountOfStockRepository amountOfStockRepository) {
+        this.orderRepository = orderRepository;
+        this.accountRepository = accountRepository;
+        this.amountOfStockRepository = amountOfStockRepository;
+    }
+    
+	
+    @Transactional
+    public OrderResponseDto createOrder(OrderRequestDto requestDto) {
```

Comment:
```suggestion
    public OrderEntity createOrder(String orderId, Int orderAmount, String stockId...) {
```
서비스레이어에서는 DTO값에 영향없이 도메인에 관련된 파라미터로 구현되는게 유지보수성에 좋습니다

Reply:
그러면
controller
OrderEntity -> OrderResponseDto
로 변경하는 from 함수 같은걸 OrderResponseDto에 추가하여 처리하여
layer별 역할을 나눌 수 있습니다